src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -75,15 +75,13 @@ def invoke_package_manager(self, command):
                                             'Patch Management cannot proceed successfully. Before the next Patch Operation, please run the following '
                                             'command and perform any configuration steps necessary on the machine to return it to a healthy state: '
                                             'sudo dpkg --configure -a')
-            self.telemetry_writer.send_execution_error(command, code, out)
```

Comment:
Retain these functions and make them capable of utilizing the telemetry channel.